packages/office-ui-fabric-react/src/components/Link/Link.scss

```diff
@@ -1,56 +1,53 @@
 @import '../../common/common';
 
-// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
-
-//
-// Office UI Fabric
-// --------------------------------------------------
-// Link (anchor) styles
-
+// When focus is set using the keyboard, apply an outline.
+@mixin focus-outline {
+  .ms-Fabric.is-focusVisible &:focus {
+    outline: 1px solid $focusedBorderColor;
+  }
+}
 
-@mixin ms-Link {
+.ms-Link {
   @include ms-baseFont;
-  @include focus-border();
   color: $ms-color-themePrimary;
-  text-decoration: none;
-  cursor: pointer;
-  padding: 0;
   margin: 0;
-  background: none;
-  @include text-align(left);
-  display: inline;
-  border: none;
-
-  // need to support ellipsis truncation, so we do this by default.
-  width: auto;
```

Comment:
I'm hesitant to change it. I think that since overflow and text-overflow is inherited that the parent would be able to easily have them both set to the necessary styles to make it work, unless I'm missing something. At least I think that was the goal.